tests/extrema_test.py

```diff
@@ -160,3 +160,15 @@ def test_error_handling(self):
         
         with self.assertRaises(ValueError) as cm:
             ak.argmaxk(ak.array([]), 1)
+
+class ArgMinTest(ArkoudaTest):
+    def test_argmin(self):
+        np_arr = np.array([False, False, True, True, False])
+        ak_arr = ak.array(np_arr)
+        self.assertTrue(np_arr.argmax() == ak_arr.argmax())
```

Comment:
i think this should be `argmin`

Reply:
Yes you are correct! this has been updated.